epub33/rs/index.html

```diff
@@ -384,14 +384,18 @@ <h3>Network access</h3>
 			<section id="sec-epub-rs-external-links">
 				<h3>External links</h3>
 
-				<p data-tests="#pub-external-links">Reading systems SHOULD open links that resolve outside the <a>EPUB
-						publication</a> in a new browser instance to ensure that the browser's security and privacy
-					controls are available to users.</p>
+				<p>Reading systems SHOULD alert users before opening hyperlinks that declare a <a
+						data-cite="url#concept-url-scheme">scheme</a> [[url]] (i.e., that resolve outside the [=EPUB
+					publication=] or require a specialized application to open).</p>
```

Comment:
I wonder about the terminology "declare a scheme"; it looks a bit strange to me. I looked at the URL spec and this terminology is not used either. What about "includes a scheme"?

I am a bit neutral about the usage of "application" vs. "user agent" (referring to the remark of @OriIdan). I have a mild preferene for user agent, which seems to be more in line with what other W3C specs are using (like UAAG).

Reply:
"has a scheme" is probably simplest.

Reply:
> user agent ... seems to be more in line with what other W3C specs are using (like UAAG)

Even UAAG defines a user agent as:

> any software that retrieves, renders and facilitates end-user interaction with web content.

A telephone number really isn't web content. That's where my disconnect comes in, as say your phone dialer gets launched, not a program that dials over the web.

Reply:
How about a hybrid statement:

> To ensure security and privacy controls are available, Reading systems SHOULD open the link in the user agent or system application registered to handle the scheme.

Reply:
Here's what's in the latest commit:

> Reading systems SHOULD alert users before opening a link that declares a [scheme](https://url.spec.whatwg.org/#concept-url-scheme) [url] (i.e., that resolves to web content outside the [EPUB publication](https://www.w3.org/TR/epub-33/#dfn-epub-publication) or requires a system application to open). If the user consents to open the link, reading systems SHOULD open the link using the default application registered for the scheme (i.e., to help ensure appropriate security and privacy controls are available).